src/tests/EventPipeTracee/Program.cs

```diff
@@ -54,12 +59,23 @@ public static int Main(string[] args)
             Console.WriteLine($"{pid} EventPipeTracee: {DateTime.UtcNow} Awaiting start");
             Console.Out.Flush();
 
+            using CustomMetrics metrics = diagMetrics ? new CustomMetrics() : null;
+
             // Wait for server to send something
             int input = pipeStream.ReadByte();
 
             Console.WriteLine($"{pid} {DateTime.UtcNow} Starting test body '{input}'");
             Console.Out.Flush();
 
+            if (diagMetrics)
+            {
+                _ = Task.Run(async () => {
+                    metrics.IncrementCounter();
+                    metrics.RecordHistogram(10.0f);
+                    await Task.Delay(10).ConfigureAwait(true);
```

Comment:
Is the delay necessary?

Reply:
I removed it now, but if you think there's a reason why you originally had it, let me know and I can put it back in.